package.json

```diff
@@ -6,7 +6,9 @@
     "doc": "docs",
     "test": "test"
   },
-  "dependencies": {},
+  "dependencies": {
+    "moment": "^2.18.1"
```

Comment:
Since we compile all the JS and include it in the repo I think we can consider this as a `devDependency`.